homeassistant/components/simplisafe/strings.json

```diff
@@ -1,15 +1,11 @@
 {
   "config": {
     "step": {
-      "input_auth_code": {
-        "title": "Finish Authorization",
-        "description": "Input the authorization code from the SimpliSafe web app URL:",
+      "user": {
+        "description": "Starting in 2021, SimpliSafe has moved to a new authentication mechanism via its web app. Due to technical limitations, there is a manual step at the end of this process; please ensure that you read the [documentation](http://home-assistant.io/integrations/simplisafe#getting-an-authorization-code) before starting.\n\n1. Click [here]({url}) to open the SimpliSafe web app and input your credentials.\n\n2. When the login process is complete, return here and input the authorization code below.",
```

Comment:
For a future PR: You might want to drop this "Starting in 2021 sentence." as we're almost 2022!

Also for a new PR: Inject the URL via a placeholder and not let translators change it.